ScorchApp/src/components/dungeonMaster/DungeonMasterInventoryManager.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div class="dm-inventory-component d-flex">
+        <div class="item-creator flex-column">
+            <div class="dropdown">
+                <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Create
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenu2">
+                    <button class="dropdown-item" type="button" v-on:click="state=1">Create Adv. Gear</button>
```

Comment:
don't use state with numbers 1,2,3,4. Give it real values like... weaponCreate or armorCreate